unitylibs/core/workflow/workflow-photoshop/workflow-photoshop.js

```diff
@@ -159,18 +174,7 @@ async function removeBgHandler(cfg, changeDisplay = true) {
     return false;
   }
   if (isImgModified) {
-    const { scanImgForSafety } = await import('../../steps/upload-step.js');
-    let scanResponse = await scanImgForSafety(cfg, id);
-    if (scanResponse.status === 403) {
-      unityEl.dispatchEvent(new CustomEvent(refreshWidgetEvent));
-      await showErrorToast(targetEl, unityEl, '.icon-error-acmp');
-      return false;
-    }
-    if (scanResponse.status === 429
-      || (scanResponse.status >= 500 && scanResponse.status < 600)) {
-      const { retryRequestUntilProductRedirect } = await import('../../../scripts/utils.js');
-      scanResponse = await retryRequestUntilProductRedirect(cfg, () => scanImgForSafety(cfg, id));
-    }
+    scanForCsam(cfg, id, unityEl, refreshWidgetEvent );
```

Comment:
nit: Remove extra space before `)`. Also, may not need `unityEl` and `refreshWidgetEvent` if we no longer intend to show any error.

Reply:
I'll fix that extra space also do you suggest we completely remove 403 (inappropriate content check) ?